assets/js/jquery-slimscroll/jquery.slimscroll.js

```
@@ -0,0 +1,474 @@
+          if (!o.alwaysVisible) { bar.hide(); }
+        }
+
+        // attach scroll events
+        attachWheel(this);
+
+        function _onWheel(e)
+        {
+          // use mouse wheel only when mouse is over
+          if (!isOverPanel) { return; }
+
+          var e = e || window.event;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'e'. Consider to delete it or rename it.

'e' is defined here:


(lint/suspicious/noRedeclare)